File: CPP/DPR.cpp
// Import library.
#include <iostream>
#include <string>

// Standard namespace.
using namespace std;

// Inisialisasi class.
class DPR
{
    // Atribut Private.
    private:
        string nama;
        string id;
        string bidang;
        string partai;

    // Atribut Public.
    public:
        // Konstruktor.
        DPR()
        {
            // Set default nilai atribut.
            this->id = "";
            this->nama = "";
            this->bidang = "";
            this->partai = "";
        }

        DPR(string nama, string id, string bidang, string partai)
        {
            // Set nilai atribut.
            this->id = id;
            this->nama = nama;
            this->bidang = bidang;
            this->partai = partai;
        }

        // Getter dan Setter.
        // Setter.
        void set_id(string id)
        {
            this->id = id;
        }
        void set_nama(string nama)
        {
            this->nama = nama;
        }
        void set_bidang(string bidang)
        {
            this->bidang = bidang;
        }
        void set_partai(string partai)
        {
            this->partai = partai;
        }

        // Getter.
        string get_id()
        {
            return this->id;
        }
        string get_nama()
        {
            return this->nama;
        }
        string get_bidang()
        {
            return this->bidang;
        }
        string get_partai()
        {
            return this->partai;
        }
        
        // Method Remove Attribute.
        void hapusDataDPR()
        {
            this->id = "";
            this->nama = "";
            this->bidang = "";
            this->partai = "";
        }

        // Destruktor
        ~DPR()
        {
            
        }
};

File: PYTHON/DPR.py
# Inisialisasi Class
class DPR:
    # Atribut DPR
    __id = ""
    __nama = ""
    __bidang = ""
    __partai = ""

    # Konstruktor
    def __init__(self, id="", nama="",  bidang="", partai=""):
        self.__id = id
        self.__nama = nama
        self.__bidang = bidang
        self.__partai = partai

    # Setter
    def set_id(self, id=""):
        self.__id = id

    def set_nama(self, nama=""):
        self.__nama = nama

    def set_bidang(self, bidang=""):
        self.__bidang = bidang

    def set_partai(self, partai=""):
        self.__partai = partai

    # Getter
    def get_id(self):
        return self.__id
    
    def get_nama(self):
        return self.__nama

    def get_bidang(self):
        return self.__bidang
        
    def get_partai(self):
        return self.__partai